instant.ts

```diff
@@ -66,9 +66,7 @@ async function runBashScript(path: string, filename: string, args: string[]) {
     }
     await promise
   } catch (err) {
-    console.error(`Error: Script ${filename} returned an error`)
-    console.log(err.stdout)
-    console.log(err.stderr)
+    process.exit(1)
```

Comment:
I think this is a little dangerous for now. Let's just log the errors properly and deal with issues as we find them. Breaking the deploy is probably going to give us headaches since it will break on failing to set elasticsearch passwords because they're already set.

Reply:
Cool, removed it in [e07cfb8](https://github.com/openhie/package-starter-kit/pull/6/commits/e07cfb8260b6dc8132ec9cec08594b8f1e570c4a), and added a little more context to the error message that was there originally.